src/Events/Authorization/XacmlMapperHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Security.Claims;
+
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Helpers;
+
+namespace Altinn.Platform.Events.Authorization
+{
+    /// <summary>
+    /// Shared methods for mapping to xacml
+    /// </summary>
+    public static class XacmlMapperHelper
+    {
+        private const string DefaultIssuer = "Altinn";
+
+        private const string UserPrefix = "/user/";
+        private const string OrgPrefix = "/org/";
+        private const string PartyPrefix = "/party/";
+
+        private const string ClaimUserId = "urn:altinn:userid";
+        private const string ClaimPartyID = "urn:altinn:partyid";
+        private const string ClaimOrg = "urn:altinn:org";
+
+        /// <summary>
+        /// Generates subject attribute list 
+        /// </summary>
+        /// <returns></returns>
+        public static XacmlJsonCategory CreateSubjectAttributes(string subjectOrResource)
```

Comment:
Should the "orResource" part of the parameter name still be there?

Reply:
nope